Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/Login.cs

```diff
@@ -56,7 +56,15 @@ protected async Task TryLogin()
 
             LoginSceneIndex = SceneManager.GetActiveScene().buildIndex;
             
-            SceneManager.LoadScene(MainSceneName);
+            // Checks if the user has removed the main scene from the build settings, allows for other packages like loot boxes to be tested with the login scene
```

Comment:
We should avoid using scene build indexes, that's why I used name to find login scene, If we have web3auth imported this will end up loading the Login scene again so lets revert this

Reply:
reverted